README.md

```diff
@@ -85,10 +85,12 @@ if (!cm.isConnected()) {
 }
 ```
 
-### onDisconnect(*callback*) ###
+### onDisconnect(*callback*[, callbackId]) ###
 
 This method assigns a callback function to the onDisconnect event. The onDisconnect event will fire every time the connection state changes from online to offline, or when ConnectionManager’s *disconnect()* method is called (even if the device is already disconnected).
 
+ConnectionManager allows for multiple onDisconnect callbacks to be registered. Each of the callbacks should have a unique string identifier passed as a second parameter to the `onDisconnect` callback setter. If the `callbackId` is not specified, a default value id value is used. Calling the setter function multiple times with the same (or omitted) `callbackId` overwrites the previously set callback.
```

Comment:
The default value should be mentioned, so the used knows not to use it.

Reply:
Also do you want to mention how to unregister a callback.

Reply:
Fixed. Thank you @betzrhodes